package.json

```diff
@@ -19,8 +19,8 @@
     "build": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\"",
     "bs": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\" \"npm run start\"",
     "quickstart": "rimraf build && concurrently \"npm run db:migrate\" \"npm run build:client\" \"npm run build:server\" \"npm run start\"",
-    "build:client": "NODE_ENV=production webpack -p --config ./webpack/prod.babel.js",
-    "build:server": "npm run build:theme && NODE_ENV=production webpack -p --config ./webpack/server.babel.js",
```

Comment:
Why not just use `npm run dev` and leave `build:client` and `build:server` alone?